### PR TITLE
Longitude/latitude displacement addition to the high-resolution radiosonde YAML

### PR DESCRIPTION
**Pull request by PraveenKumar-NOAA**
## Description

This PR adds the longitude/latitude displacement and WIGOSID to the existing high-resolution radiosonde YAML, which is required by both global and regional users, and also replaces dewpointTemperature with dewPointTemperature.

### Issue(s) addressed

This PR partially addresses the following issue #1184. 

## Acceptance Criteria (Definition of Done)

Good reviews and pass the ctest.

## Dependencies

None

## Impact

None

### CODE REVIEW

**Review by MatthewMorris-NOAA — APPROVED** [review]
Looks good.

File: test/testinput/bufr_ncep_highRes_sonde.yaml [hunk 1]
```diff
@@ -26,17 +26,23 @@ observations:
             query: "*/RPID"
           latitude:
             query: "*/CLATH"
+          levelLatitudeDisplacement:
+            query: "*/UARLVB/LATDH"
```
Comment (BenjaminRuston) [inline test/testinput/bufr_ncep_highRes_sonde.yaml:30]: what are they using for GNSS-RO, this too has a single value and then individual ones for each level.  the code is designed for a displacement from a single value (as this is pretty specific to a data type)? Or should the converter compute the lat/lon at each vertical point and store this instead.

@emilyhcliu do you have any input on this?

**Review by BenjaminRuston — CHANGES_REQUESTED** [review]
can we add level by level latitude and longitude like we do for other observation types such as `gnss_ro` rather than add a new convention for `levelLatitudeDisplacement`

@emilyhcliu ?

**Comment by PraveenKumar-NOAA** [comment]
> can we add level by level latitude and longitude like we do for other observation types such as `gnss_ro` rather than add a new convention for `levelLatitudeDisplacement`
> 
> @emilyhcliu ?

@BenjaminRuston this is required by the regional RTMA/URMA users and @MatthewMorris-NOAA is the right person to answer this. I am not adding a new convention here, this already exists into the conventional table.

**Comment by emilyhcliu** [comment]
> can we add level by level latitude and longitude like we do for other observation types such as `gnss_ro` rather than add a new convention for `levelLatitudeDisplacement`
> 
> @emilyhcliu ?

@BenjaminRuston, what is the official name of the gnss_ro (level by level lat/lon) data you mention in the IODA naming convention?  If the proper variable name for radiosonde data displacement already exists, we have no objection to adopting the name you suggest.

**Comment by PraveenKumar-NOAA** [comment]
@BenjaminRuston @MatthewMorris-NOAA @emilyhcliu @PatNichols please check the [WMO Table](https://confluence-test.ecmwf.int/display/ECC/WMO%3D35+element+table#WMO=35elementtable-CL_5), the displacement data is defined as: 

005015 --- latitudeDisplacement
006015 --- longitudeDisplacement

I am using the variables already in the [ObsSpace.yaml](https://github.com/JCSDA-internal/ioda/blob/develop/share/ioda/yaml/validation/ObsSpace.yaml#L1227). 

I also extracted values of these variables directly from the BUFR file, I hope this will help: 

005015  LATDH                    0.58870  DEGREE                        LATITUDE DISPLACEMENT (HIGH ACCURACY)
005015  LATDH                    0.58988  DEGREE                        LATITUDE DISPLACEMENT (HIGH ACCURACY)
005015  LATDH                    0.59110  DEGREE                        LATITUDE DISPLACEMENT (HIGH ACCURACY)

006015  LONDH                    1.66105  DEGREE                        LONGITUDE DISPLACEMENT (HIGH ACCURACY)
006015  LONDH                    1.66371  DEGREE                        LONGITUDE DISPLACEMENT (HIGH ACCURACY)
006015  LONDH                    1.66650  DEGREE                        LONGITUDE DISPLACEMENT (HIGH ACCURACY).

**Comment by emilyhcliu** [comment]
@MatthewMorris-NOAA Thanks for the confirmation.  We will ensure the global file contains the actual (true) location (lat/lon) of the observations.

**Comment by emilyhcliu** [comment]
@BenjaminRuston We will drop the displacement data from the IODA file.  And, we will add the processing to create the actual lat/lon from the lunch point lat/lon plus the displacement in the bufr converter.

**Review by emilyhcliu — APPROVED** [review]

**Review by MatthewMorris-NOAA — APPROVED** [review]

**Review by BenjaminRuston — APPROVED** [review]
excellent thanks!

**Pull request merged**